src/python/pants/backend/experimental/python/lint/ruff/register.py

```diff
@@ -15,6 +15,6 @@ def rules():
         "2.23.0.dev0",
         "using the `pants.backend.experimental.python.lint.ruff` backend",
         hint="Use `pants.backend.experimental.python.lint.ruff.check` instead.",
-        start_version="2.21.0.dev0",
+        start_version="2.20.0.dev0",
```

Comment:
Note - I changed this so that I could actually get the message to show up when running locally. And it does!

```bash
PANTS_SOURCE=~/Projects/pants pants fix lint ::

Pantsd has been turned off via Env.
08:29:19.36 [WARN] DEPRECATED: using the `pants.backend.experimental.python.lint.ruff` backend is scheduled to be removed in version 2.23.0.dev0.

Use `pants.backend.experimental.python.lint.ruff.check` instead.
```

I think probably good to leave this at `2.20.0.dev0` which has already been released, versus `2.21.0.dev0` which might be far into the future.